df_py/util/blocktime.py

```diff
@@ -120,7 +120,7 @@ def timeSinceTimestamp(self, block_i):
                 block_timestamp = web3.eth.get_block(int(block_i)).timestamp
             except Exception as e:
                 print(f"An exception occurred while getting block {block_i}, {e}")
-                return 0
+                block_timestamp = 0
             return block_timestamp - self.target_timestamp
```

Comment:
I see now that it's ok for this to return `-self.target_timestamp` in the case of an Exception, because it's handled below.